pkg/alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # image was bootstraped using FROM lfedge/eve-alpine-base:fad44e3702708a8d044663a20fd98d933dddb41e AS cache
 # to update please see https://github.com/lf-edge/eve/blob/master/docs/BUILD.md#how-to-update-eve-alpine-package
-FROM lfedge/eve-alpine:1f7685f95a475c6bbe682f0b976f12180b6c8726 AS cache
+FROM lfedge/eve-alpine:fbf39533415522ac124a2da9a695fda686adedc3 AS cache
```

Comment:
Shouldn't we update all of the FROM eve-alpine lines to use the same tag?

Reply:
If we go ahead with this. I'm planning on leaving this open, as the eve-rust approach is better. Once that is in, I'll close this.